pkg/dotc1z/grants.go

```diff
@@ -62,6 +62,29 @@ func (r *grantsTable) Migrations(ctx context.Context, db *goqu.Database) error {
 	return nil
 }
 
+// DropGrantIndexes drops the indexes on the grants table.
+// This should only be called when compacting the grants table.
+// These indexes are re-created when we open the database again.
+func (c *C1File) DropGrantIndexes(ctx context.Context) error {
+	ctx, span := tracer.Start(ctx, "C1File.DropGrantsIndexes")
+	defer span.End()
+
+	indexes := []string{
+		fmt.Sprintf("idx_grants_resource_type_id_resource_id_v%s", grants.Version()),
```

Comment:
wish these were constants/otherwise available in the places generating them and now.. regenerating them.

Reply:
Fixed in https://github.com/ConductorOne/baton-sdk/pull/572/commits/de28762f80e3961eec1e23e34c814544c602e346